content/en/admin/troubleshooting.md

```diff
@@ -30,3 +30,13 @@ Check that you are specifying the correct environment with `RAILS_ENV=production
 ## **I encountered a compilation error while executing `RAILS_ENV=production bundle exec rails assets:precompile`, but no more information is given. How to fix it?**
 
 Usually it's because your server ran out of memory while compiling assets. Use a swapfile or increase the swap space to increase the memory capacity. Run `RAILS_ENV=production bundle exec rake tmp:cache:clear` to clear cache, then execute `RAILS_ENV=production bundle exec rails assets:precompile` to compile again. Make sure you clear the cache after a compilation error, or it will show "Everything's OK" but leave the assets unchanged.
+
+## **When using an external file system for the live folder, the message appears - `Read-only file system @ dir_s_mkdir`. How to fix it?**
```

Comment:
I would say “When installing Mastodon on an external file system, the following message appears […]”